tests/cypress/tests-e2e/legacyParamImport.cy.js

```diff
@@ -411,4 +411,55 @@ describe('Test on legacy param import', () => {
             cy.readStoreValue('state.position.projection.epsgNumber').should('eq', 3857)
         })
     })
+
+    context('Bod Layer Id import', () => {
+        function checkFeatures(featuresIds) {
+            cy.readStoreValue('state.features.selectedFeatures').then((features) => {
+                cy.wrap(features.length).should('be.equal', featuresIds.length)
+                features.forEach((feature) => {
+                    cy.wrap(featuresIds.includes(feature._id)).should('be.true')
+                })
+            })
+        }
+
+        beforeEach(() => {
+            // add intercept for layer config
+            cy.intercept('**/rest/services/all/MapServer/layersConfig**', {
+                fixture: 'layers.fixture',
+            })
+            // add intercept for the html popup
+            cy.intercept('**/MapServer/**/htmlPopup**', {
+                fixture: 'html-popup.fixture.html',
+            })
+        })
+
+        describe('Checks that the legacy bod layer id translate in the new implementation', () => {
+            it('Select a few features and shows the tooltip in its correct spot', () => {
+                const features = []
+                const nbFeatures = 4
+                cy.fixture('features.fixture.json').then((results) => {
+                    results['results'].forEach((feature) => {
+                        features.push(feature)
+                    })
+                    const layer = features[0].layerBodId
+                    for (let i = 0; i < nbFeatures; i++) {
+                        // we intercept every feature we want to retrieve
+                        cy.intercept(`**/MapServer/${layer}/${features[i].id}`, {
+                            results: [features[i]],
+                        })
+                    }
+
+                    const featuresIds = []
+
+                    for (let i = 0; i < nbFeatures; i++) {
+                        featuresIds.push(features[i].id.toString())
+                    }
+                    const params = {}
+                    params[layer] = featuresIds.join(',')
```

Comment:
Isn't the current (legacy) syntax doing something like
`https://.../?layer.id=1&layer.id=2&layer.id=3` instead of `https://.../?layer.id=1,2,3` ?